packages/components/src/styles/components/code-editor.scss

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// CODE-EDITOR
+//
+
+.hds-code-editor {
+  $border-strong: rgba(178, 182, 189, 40%);
+
+  background-color: var(--token-color-code-surface-primary);
+  border: 1px solid;
+  border-radius: 6px;
+
+  &--is-fullscreen {
```

Comment:
[todo] we don't use this kind of nesting in HDS (on purpose), we prefer a much flatten structure, and avoid using the `&` operator to create class names, in favour of more explicit (and easy to find) names

Reply:
For sure. This was just shorthand for me during development.